spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -45,14 +45,53 @@ interface FungibleTokenPacketData {
   receiver: string
   memo: string
 }
+
+interface FungibleTokenPacketDataV2 {
+  tokens: []Token
+  sender: string
+  receiver: string
+  memo: string
+}
+
+interface Token {
+  denom: string
+  trace: []string
+  amount: uint64
+  metadata: Metadata
+}
+
+interface Metadata {
+  denomUnits: []DenomUnit
+}
+
+interface DenomUnit {
+  denom: string
+  exponent: uint256
+}
 ```
 
+As tokens are sent across chains using the ICS 20 protocol, they begin to accrue a record of channels for which they have been transferred across. This information is encoded into the `trace` field in the denomination. 
```

Comment:
As tokens are sent across chains using the ICS 20 protocol, they begin to accrue a record of channels for which they have been transferred across. This information is encoded into the `trace` field and will be used in tandem with the base denomination, by the `constructOnChainDenom` helper function, to construct the local on-chain denomination for the bridged token. 
```suggestion
As tokens are sent across chains using the ICS 20 protocol, they begin to accrue a record of channels for which they have been transferred across. This information is encoded into the `trace` field and will be used in tandem with the base denomination, by the `constructOnChainDenom` helper function, to construct the local on-chain denomination for the bridged token.
```